src/OnlySpans.PolyLeads.Api.Schema/Payloads/Documents/RecognitionStatus.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace OnlySpans.PolyLeads.Api.Schema.Payloads.Documents;
+
+[GraphQLDescription(
+    """
+    Статус распознавания текста:
+    0 - неизвестен
+    1 - в очереди
+    2 - выполняется
+    3 - текст успешно распознан
+    4 - при распозновании текста произошла ошибка
+    """)]
+public enum RecognitionStatus
+{
+    Unknown,
+    Queued,
+    Processing,
+    Success,
+    Error
```

Comment:
на значения enum можно тоже вешать атрибуты
```suggestion
public enum RecognitionStatus
{
   [GraphQLDescription("Неивзестен")]
    Unknown,

    [GraphQLDescription("В очереди")]
    Queued,
    Processing,
    Success,
    Error
```